### PR TITLE
Switching to ubicloud arm runner

### PR DESCRIPTION
**Pull request by Maleware**
Adds ubicloud-standard-8-arm to operator build pipeline

### CODE REVIEW

File: template/.actionlint.yaml [hunk 1]
```diff
@@ -2,5 +2,4 @@
 self-hosted-runner:
   # BuildJet machines we are using
```
Comment (razvan) [inline template/.actionlint.yaml:3]: I guess this comment is now wrong

**Review by razvan — APPROVED** [review]
see comment

File: template/.actionlint.yaml (hunk 1 above)
Comment (Maleware) [reply template/.actionlint.yaml]: Thank you! Good catch :)

**Comment by Maleware** [comment]
https://github.com/stackabletech/opa-operator/actions/runs/9596294742

tested successfully.

**Review by razvan — APPROVED** [review]
lgtm!

**Pull request merged**